ui/README.md

```diff
@@ -0,0 +1,64 @@
+# UI
+
+This directory is for the React (non-lite/AMP) UI.
+
+## Development server
+
+The easiest way to develop the UI is to run two local servers:
+
+1. Run the Django (backend) server as usual, at port 8000.
+2. Run the UI with `tools/ui run`. It will talk to the backend at port 8000.
+
+Changes you make to the JS and CSS will take effect when you refresh the page.
+If you make other changes (e.g., to install a new module or modify the Webpack
+config) you'll likely need to restart the server.
+
+Currently, it is necessary to change `USE_REACT_UI` in `app/main.py` to `True`.
+Please be careful not to commit this change.
+
+## Deployment
+
+You can run `tools/ui buildtoae` to compile the React app into language-specific
```

Comment:
What do you mean by language-specific? Is it natural language or programming language?

Reply:
Natural language. I changed the doc to clarify it.